dns-policy-operator/src/charm.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python3
+
+"""DNS policy charm."""
+
+import json
+import logging
+import pathlib
+import time
+import typing
+
+import ops
+from charms.bind.v0 import dns_record
+from charms.data_platform_libs.v0.data_interfaces import (
+    DatabaseCreatedEvent,
+    DatabaseEndpointsChangedEvent,
+)
+from charms.operator_libs_linux.v1 import systemd
+
+import constants
+import models
+import templates
+from database import DatabaseHandler
+from dns_policy import DnsPolicyService
+
+# Log messages can be retrieved using juju debug-log
+logger = logging.getLogger(__name__)
+
+VALID_LOG_LEVELS = ["info", "debug", "warning", "error", "critical"]
+
+
+class ReconciliateEvent(ops.charm.EventBase):
```

Comment:
Shouldn't it be "Reconcile" or "Reconciliation" ?

Reply:
Fixed